tests/src/main/scala/scalaz/tests/IListTests.scala

```diff
@@ -22,93 +22,272 @@ final class IListTests {
   def cross[A, B](l1: List[A], l2: List[B]): List[(A, B)] =
     l1.flatMap(a1 => l2.map(a2 => (a1, a2)))
 
-  def tests[T](harness: Harness[T]): T = {
+  def tests[T](harness: Harness[T], sequence: (T, T) => T): T = {
     import harness._
-    section("laws")(
-      section("eq laws")(
-        test("reflexivity") { () =>
-          lists.foldMap(
-            EqLaws.reflexivity(_)(assert)
-          )
+    sequence(
```

Comment:
Why is this necessary?

Reply:
Two sections at the same level with no parent section.